earth_enterprise/src/SConstruct

```diff
@@ -97,6 +98,7 @@ coverage = ARGUMENTS.get('coverage', 0)
 bundle_3rd_libs = int(ARGUMENTS.get('bundle_3rd_libs', 0))
 cc_dir = ARGUMENTS.get('cc_dir', None)
 log_performance = int( ARGUMENTS.get( 'log_performance', 0 ) )
+cpp_standard = ARGUMENTS.get('cpp_standard', 'gnu++11')
```

Comment:
Well, on very new distros with gcc 6 or later, c++11 is the default, not gnu++11.  Is it possible to test gcc versions from scons?

Reply:
Yes we need to move away from gnu extensions.  The good news is I believe GEE code doesn't rely on any extensions directly but some of the 3rd party libraries do.  I'm hoping the effort to update to newer 3rd party libraries will allow us to then switch to c++11.

Reply:
BTW we don't compile with c++98 either...  We need use gnu++98.  I think the dependency on gnu extensions is a different issue that should be raised and addressed outside this issue.

Reply:
Created #775